modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/js/config.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+// eslint-disable-next-line no-undef
+Liferay.Loader.require(MODULE_MAIN, (_) => {});
```

Comment:
Can you remind me again what this is for? I looked through our existing uses of `Liferay-JS-Config:` and the corresponding files, and they typically all point to a file that adds it to the AUI module loader.

See these for example [one](https://github.com/liferay/liferay-portal/blob/42e897766c504434afa77745fb6afd8e69d566d2/modules/apps/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/config.js), [two](https://github.com/liferay/liferay-portal/blob/42e897766c504434afa77745fb6afd8e69d566d2/modules/apps/bookmarks/bookmarks-web/src/main/resources/META-INF/resources/bookmarks/js/config.js), [three](https://github.com/liferay/liferay-portal/blob/42e897766c504434afa77745fb6afd8e69d566d2/modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/config.js)

Reply:
The idea here is only to trigger the init of the module which causes OAuth2Client to bind to `Liferay.OAuth2Client` so that it's globally available.

Reply:
If you have another way to achieve this please let me know, I did ask many times how to accomplish this but no-one seemed to get what I was trying to do.

The end result is that one should be able to execute this bit of JS from anywhere in the frontend:
```javascript
  var theProfileExternalReferenceCode = ...

  Liferay.OAuth2Client.FromUserAgentApplication(
    theProfileExternalReferenceCode
  ).fetch(
    '/some/path'
  ).then(res => {
    ...
  });
```

Reply:
I think you can use `Liferay-JS-Resources-Top-Head` instead of `Liferay-JS-Config` and it will have the same effect (but would be, semantically, more correct). See [example](https://github.com/liferay/liferay-portal/blob/19ea4424314059cdb2447c033cd87ae886a3b7ee/modules/apps/frontend-js/frontend-js-web/bnd.bnd#L4).